File: AniDAAPI/AniDAAPI/Models/CustPerShipLU.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace AniDAAPI.Models
{
    public partial class CustPerShipLU
    {
        public int CustPersonShipLUId { get; set; }
        public int? CustomerId { get; set; }
        public int? CustPersonId { get; set; }
        public int? VesselId { get; set; }
        public int? LastUpdateBy { get; set; }
        public DateTime? LastUpdatedDt { get; set; }
    }
}

File: login_and_signup_web-master/lib/Model/CustPerShipLU.dart
class CustPerShipLU {
  int custPerShipLUID;
  int customerID;
  int custPersonID;
  int vesselID;
  int lastUpdatedBy;
  String lastUpdatedDT;

  CustPerShipLU({
    this.custPerShipLUID,
    this.customerID,
    this.custPersonID,
    this.vesselID,
    this.lastUpdatedBy,
    this.lastUpdatedDT,
  });

  factory CustPerShipLU.fromJson(Map<String, dynamic> json) {
    return CustPerShipLU(
      custPerShipLUID: json['custPerShipLUId'],
      customerID: json['customerId'],
      custPersonID: json['custPersonId'],
      vesselID: json['vesselId'],
      lastUpdatedBy: json['lastUpdatedBy'],
      lastUpdatedDT: json['lastUpdatedDt'],
    );
  }
}
